website/blog/2020-02-07-announcing-riff-0-5-0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Announcing riff v0.5.0"
+---
+
+We are happy to announce riff v0.5.0. Thanks to all riff, Cloud Native Buildpack, Knative, Liiklus, Keda, Contour, and other contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/cli/releases/tag/v0.5.0) on GitHub. The [getting started](/docs/v0.5/getting-started) guides will help you to run your first function.
+
+![Cambridge UK bridge](assets/cam-bridge.jpg)
+
+## Streaming
+
+This release introduces a new experimental streaming capability. Functions deployed using the [Streaming Runtime](/docs/v0.5/runtimes/streaming) are able to consume and produce streams of events. The Node and Java function invokers have been extended to support a new [streaming invoker specification](https://github.com/projectriff/invoker-specification/blob/master/streaming.md).
+
+Here is an example of a JavaScript streaming function interface with 2 input and 1 output stream, using the [node function invoker]().
+```js
+module.exports = (inputStreams, outputStreams) => {
```

Comment:
if you want to be more fancy:

```js
module.exports = ({ impressions, orders }, { conversions }) => {
     // do something
 };
 module.exports.$interactionModel = 'node-streams';
```

Reply:
That cool - i think we should stick with the slightly longer version for clarity.